ci/workflows/pull-request.yml

```diff
@@ -43,270 +43,18 @@ steps:
     timeout_in_minutes: 15
     retry: *auto-retry
 
-  - label: "build other components"
-    command: "ci/scripts/build-other.sh"
-    key: "build-other"
+  - label: "S3 source check on AWS (csv parser)"
+    command: "ci/scripts/s3-source-test.sh -p ci-dev -s run_csv"
+    depends_on: build
     plugins:
       - seek-oss/aws-sm#v2.3.1:
           env:
-            GITHUB_TOKEN: github-token
+            S3_SOURCE_TEST_CONF: ci_s3_source_test_aws
       - docker-compose#v4.9.0:
           run: rw-build-env
           config: ci/docker-compose.yml
           mount-buildkite-agent: true
           environment:
-            - GITHUB_TOKEN
-    timeout_in_minutes: 10
-    retry: *auto-retry
-
-  - label: "build (deterministic simulation)"
-    command: "ci/scripts/build-simulation.sh"
-    key: "build-simulation"
-    plugins:
-      - gencer/cache#v2.4.10: *cargo-cache
```

Comment:
deleted workflow?

Reply:
Just for testing my new workflow, without waiting others. I will move it to main-cron once it becomes green.